services/121-service/src/config.ts

```diff
@@ -1,5 +1,3 @@
-import packageJson = require('../package.json');
```

Comment:
I made this change because somewhere it was complaining about this `require`. I figured we weren't missing out on much without it.